lib/zip/extra_field.rb

```diff
@@ -3,7 +3,7 @@ class ExtraField < Hash
     ID_MAP = {}
 
     def initialize(binstr = nil)
-      binstr and merge(binstr)
+      binstr && merge(binstr)
```

Comment:
@simonoff Can you help me to simplify this statement without usage of 'and', I read http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby, but still not quite sure about it